examples/flutter_gallery/lib/demo/material/full_screen_dialog_demo.dart

```diff
@@ -164,19 +165,41 @@ class FullScreenDialogDemoState extends State<FullScreenDialogDemo> {
           children: <Widget>[
             new Container(
               padding: const EdgeInsets.symmetric(vertical: 8.0),
-              decoration: new BoxDecoration(
-                border: new Border(bottom: new BorderSide(color: theme.dividerColor))
-              ),
               alignment: Alignment.bottomLeft,
-              child: new Text('Event name', style: theme.textTheme.display2)
+              child: new TextField(
+                decoration: const InputDecoration(
+                  labelText: 'Event name',
+                  hintText: 'What is the name of the event?',
```

Comment:
This text doesn't always fit, on my Nexus 5 it becomes "What is the name of the...". Since the labelText makes the field's purpose pretty clear; best to leave out the hintText.